packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -336,6 +336,14 @@ export interface GridOptions {
    * You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository.
    */
   localeText: Partial<GridLocaleText>;
+  /**
+   *
+   */
+  onScrollBottom?: () => void;
```

Comment:
onRowsScrollEnd?
You will need to add the params

Reply:
I'll pass in the event itself as there isn't a need for a specific set of grid related params.

Reply:
Well that will be the second part. You will need to pass, the index of the last loaded rows at least. 
Also the viewport page size could be useful.
The visible columns as well, if they use graphQL they can load only the visible data.
The api as always. 
We could also argue about the filter and sort model so the server can apply those too 🤔

Reply:
Done, I added `viewportPageSize`, `virtualRowsCount`, `visibleColumns` and the `api`. Those seemd to help with the fetching of data